src/components/SearchResults/SearchResults.tsx

```diff
@@ -29,7 +29,7 @@ const SearchResults = (props: SearchResultsProps) => {
     items: hits,
   });
   const { currentPageItems, currentPage } = paginationProps;
-
+  console.log(hits, "hits");
```

Comment:
Noticed this console log in the console.

Looks good on the site!

Reply:
removed